### PR TITLE
Unity installation changes for central helm

### PR DESCRIPTION
**Pull request by karthikk92**
# Description
Unity installation changes for central helm

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/877 |

# Checklist:

- [ ] Have you run a grammar and spell checks against your submission?
- [ ] Have you tested the changes locally?
- [ ] Have you tested whether the hyperlinks are working properly?
- [x] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

File: content/docs/csidriver/installation/helm/unity.md [hunk 1]
```diff
@@ -107,13 +107,17 @@ Procedure
       * ArrayId corresponds to the serial number of Unity XT array.
       * Unity XT Array username must have role as Storage Administrator to be able to perform CRUD operations.
       * If the user is using a complex K8s version like "v1.24.6-mirantis-1", use this kubeVersion check in helm/csi-unity/Chart.yaml file.
-            kubeVersion: ">= 1.24.0-0 < 1.28.0-0"
+            kubeVersion: ">= 1.24.0-0 < 1.29.0-0"
 
-2. Copy the `helm/csi-unity/values.yaml` into a file named `myvalues.yaml` in the same directory of `csi-install.sh`, to customize settings for installation.
+2. Get the required values.yaml using below command
```
Comment (sharont58) [inline content/docs/csidriver/installation/helm/unity.md:112]: using the command below
or 
using the following command

**Review by sharont58 — CHANGES_REQUESTED** [review]
See comment

File: content/docs/csidriver/installation/helm/unity.md (hunk 1 above)
Comment (karthikk92) [reply content/docs/csidriver/installation/helm/unity.md]: Taken care

**Comment by karthikk92** [comment]
> See comment

Taken care

**Review by rajkumar-palani — DISMISSED** [review]
lgtm

File: content/docs/csidriver/installation/helm/unity.md [hunk 2]
```diff
@@ -107,13 +107,17 @@ Procedure
...
 
-3. Edit `myvalues.yaml` to set the following parameters for your installation:
+```bash
+cd dell-csi-helm-installer && wget -O my-unity-settings.yaml -b <version> https://raw.githubusercontent.com/dell/helm-charts/main/charts/csi-unity/values.yaml
+```
+
+3. Edit `values.yaml` to set the following parameters for your installation:
    
-    The following table lists the primary configurable parameters of the Unity XT driver chart and their default values. More detailed information can be found in the [`values.yaml`](https://github.com/dell/csi-unity/blob/master/helm/csi-unity/values.yaml) file in this repository.
+    The following table lists the primary configurable parameters of the Unity XT driver chart and their default values. More detailed information can be found in the [`values.yaml`](https://raw.githubusercontent.com/dell/helm-charts/main/charts/csi-unity/values.yaml) file in this repository.
```
[the remark this answers is not in the document]
Comment (karthikk92) [reply content/docs/csidriver/installation/helm/unity.md]: Taken care

**Review by rensyct — APPROVED** [review]

**Review by rajkumar-palani — APPROVED** [review]
lgtm

File: content/docs/csidriver/installation/helm/unity.md (hunk 1 above)
Comment (karthikk92) [reply content/docs/csidriver/installation/helm/unity.md]: @sharont58  Could you please review and approve this PR ?

**Review by sharont58 — APPROVED** [review]

**Pull request merged**